PF-session1/assignment3/TestLongestSequence.java

```diff
@@ -0,0 +1,25 @@
+package assignment3;
+
+import static org.junit.Assert.*;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class TestLongestSequence 
+{
+	LongestSequence test;
+	int arr[],expectedArr[];
+	@Before
```

Comment:
@Before is useful when setting up common data between multiple unit tests. Here, there's only one unit test. So why do we need @Before?

Also, note that even if there are multiple unit tests, use of @Before will fail in this case because unit tests could potentially be run in multi-threaded environment, which will cause problems in our test because we are instantiating common data structures (arr[] and expectedArr[]) in the "before" method.